lib/runners/processor/pmd_cpd.rb

```diff
@@ -0,0 +1,199 @@
+module Runners
+  class Processor::PmdCpd < Processor
+    include Java
+
+    Schema = StrongJSON.new do
+      let :runner_config, Schema::BaseConfig.base.update_fields { |fields|
+        fields.merge!({
+                    minimum_tokens: numeric?,
```

Comment:
[suggestion] How about using `-` instead of `_`? Original CLI flags use `-`, so I think users are less confused.

```suggestion
                    minimum-tokens: numeric?,
```

Ruby code:

```diff
-config_linter[:minimum_tokens]
+config_linter[:"minimum-tokens"]
```

Other flags are the same.

Reply:
Good call!
Fixed. commit 2b3488b8aa47595daf381b39e1bf9aa2d9a13dbb